cluster/lib/logging.sh

```diff
@@ -114,6 +120,11 @@ kube::log::usage_from_stdin() {
 
 # Print out some info that isn't a top level status line
 kube::log::info() {
+  local V="${V:-0}"
+  if [[ $KUBE_VERBOSE < $V ]]; then
```

Comment:
of course, sorry, I mis-parsed it, I was expecting to see >=, so I missed
tha this was the opposite case.  LGTM

On Thu, Aug 4, 2016 at 9:58 PM, Tim St. Clair <notifications@github.com>
wrote:

> In cluster/lib/logging.sh
> <https://github.com/kubernetes/kubernetes/pull/30115#discussion_r73642032>
> :
>
> > @@ -114,6 +120,11 @@ kube::log::usage_from_stdin() {
> >
> >  # Print out some info that isn't a top level status line
> >  kube::log::info() {
> > +  local V="${V:-0}"
> > +  if [[ $KUBE_VERBOSE < $V ]]; then
>
> I was trying to match the behavior of glog, which is to log if the
> verbosity level is >= the log level.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/30115/files/a330930d416f6afd69d402ab009716c708940f90#r73642032>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVLFwPioQPfXmp5enPMpaEI3LmcsVks5qcsL8gaJpZM4JdQg0>
> .
>